contentcuration/contentcuration/frontend/channelEdit/components/sidePanels/SubmitToCommunityLibrarySidePanel/index.vue

```diff
@@ -196,47 +219,54 @@
       LoadingText,
       StatusChip,
       CountryField,
+      KCheckbox,
```

Comment:
Here, we don't need to declare the KCheckbox component because it is already declared as part of the KDS installation [here](https://github.com/learningequality/kolibri-design-system/blob/2a74698fdc07c6d3d4eb3f173b4911a1648bf0ab/lib/KThemePlugin.js#L131)